api/models/dishesModel.js

```diff
@@ -0,0 +1,45 @@
+const db = require('../../data/dbConfig.js')
+
+module.exports = {
+    find,
+    findById,
+    findRecipesById,
+    add,
+    update,
+    remove
+}
+
+function find() {
+    return db('dishes')
+}
+
+function findById(id) {
+    return db('dishes')
+        .where({ id })
+        .first()
+}
+
+function findRecipesById(dishId) {
+    return db('recipes')
+        .join('dishes', 'recipes.dish_id', 'dishes.id')

```

Comment:
So this works. But the join is unnecessary because there's no join table (like dishes_recipes). 
`db('recipes').where('dish_id', dishId)` would work fine. I'm not huge on the naming convention here, I know this is in the dish model and it makes sense, but I would go with something like findDishesRecipes